packages/ts-types/README.md

```diff
@@ -83,6 +84,33 @@ The important things to note are:
 - `"types": ["@openui5/ts-types"]` to help the TypeScript based language service resolve the global
   type definitions from the @openui5/ts-types package.
 
+#### UI5 development
+If you want to create ts files and let typescript generate appropriate js files your [tsconfig.json][ts-config] could look like this 
+
+```json
+{
+  "compilerOptions": {
+    "module": "none",
+    "target": "es5",
+		"sourceMap": true,
+    "noEmit": false,
+    "checkJs": false,
+    "allowJs": false,
+    "types": ["@openui5/ts-types"]
+  }
+}
+```
+
+The important things to note are:
+
+- `"module": "none"` because UI5 has its own unique modules system.
+- `"target": "es5"` let typescript generate es5 files to be able to run on older browsers
+- `"sourceMap": true` let typescript generate sourceMap files to enable browsers to debug your ts files
+- `"noEmit": false` as we want typescript to generate code.
```

Comment:
we can remove "noEmit" and "checkJS" and "allowJS" because by default they are false

Reply:
We probably want to add an "outDir" like in the demo-ts example